sktime/annotation/e_agglo.py

```diff
@@ -280,6 +285,40 @@ def get_cluster(estimates):
 
         return self
 
+    def _transform(self, X, y=None):
+        """Transform X and return a transformed version.
+        
+        private _transform containing core logic, called from transform
+
+        Parameters
+        ----------
+        X : Series of mtype X_inner_mtype
+            Data to be transformed
+        y : Series of mtype y_inner_mtype, default=None
+            Not required for this unsupervised transform.
+
+        Returns
+        -------
+        transformed version of X, representing the original data on a new set of
+        coordinates, obtained by multiplying input data by the basis vectors.
+        """
+        # fit again if indices not seen, but don't store anything
+        if not X.index.equals(self._X.index):
+            X_full = X.combine_first(self._X)
+            new_eagglo = EAGGLO(
+                member=self.member,
+                alpha=self.alpha,
+                penalty=self.penalty,
+            ).fit(X_full)
+            warnings.warn(
+                "Warning: Input data X differs from that given to fit(). "
+                "Refitting with new input data, not storing updated public class "
```

Comment:
Should say you refit with *both* the data in fit and new input data. Current wording suggests fitting with only new input data.